src/omnibar/omnibar.spec.ts

```diff
@@ -329,10 +329,13 @@ describe('Omnibar', () => {
   });
 
   describe('interop with host page', () => {
-    it('should ignore messages that do not originate from omnibar', () => {
-      messageIsFromOmnibarReturnValue = false;
 
+    beforeEach(() => {
```

Comment:
Putting the `loadOmnibar()` call in `beforeEach()` changes the order of when `loadOmnibar()` is called in tests, and it also causes it to be called twice in some tests. While this may not be a problem now, it could lead to hard-to-debug issues later. I suggest putting the `loadOmnibar()` calls back in each test where it was removed.